PSD-Sign-up_page.html

```diff
@@ -66,7 +67,7 @@
 }
 </style>
 <body>
-
```

Comment:
Try to make the code in small pieces which will be easy to understand for other developers.